log.go

```diff
@@ -17,6 +17,15 @@ func WriteLog(message string) {
 		os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
 
 	defer f.Close()
+	
+	if message == "\n" {
+		
```

Comment:
@arjundashrath can you please add a comment to explain why this is being done?

Reply:
Done